pkg/cli/cmds/server.go

```diff
@@ -69,7 +70,7 @@ var (
 		"with-node-id":                drop,
 		"node-label":                  copy,
 		"node-taint":                  copy,
-		"docker":                      copy,
+		"docker":                      drop,
```

Comment:
This seems to be addressing an unrelated issue. If this does need to be "fixed" I'd prefer that to be in a separate pull-request. Or, at the very least, link to the issue this addresses in this pull-request and call it out in the commit comment.

Reply:
I don't believe rke2 supports docker as a backend, does it? I'll create an issue if there isn't one.

Reply:
Opened #365, will remove this from here.